pkgs/development/python-modules/mysql-connector/default.nix

```
@@ -31,6 +42,6 @@ in buildPythonPackage rec {
     homepage = "https://github.com/mysql/mysql-connector-python";
     changelog = "https://raw.githubusercontent.com/mysql/mysql-connector-python/${version}/CHANGES.txt";
     license = [ lib.licenses.gpl2Only ];
-    maintainers = with lib.maintainers; [ ];
+    maintainers = with lib.maintainers; [ turion neosimsim ];
```

```suggestion
    maintainers = with lib.maintainers; [
      neosimsim
      turion
    ];
```
I vaguely remember that some people prefer this style.